converter/converters/pod/affinity.go

```diff
@@ -0,0 +1,398 @@
+package pod
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+	"github.com/koki/short/types"
+)
+
+func RevertAffinities(a []types.Affinity) (*v1.Affinity, error) {
```

Comment:
func revertAffinities(affinities []types.Affinity) (*v1.Affinity, error)

Reply:
Move this method into the existing file, like the rest of the transforms.

Reply:
The main Pod file was over 1k lines, so I put the Affinity code in an analogous file `koki_affinity_to_kube_v1_affinity.go`. I think it's worthwhile to keep file lengths down.